src/lib/assets/index.ts

```diff
@@ -28,6 +29,68 @@ import { watcher, moduleSecret } from "./pods";
 
 import { clusterRoleBinding, serviceAccount, storeRole, storeRoleBinding } from "./rbac";
 import { createDirectoryIfNotExists } from "../filesystemService";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function toYaml(obj: any): string {
+  return dumpYaml(obj, { noRefs: true });
+}
+
+function webhookYaml(
+  assets: Assets,
+  whc: kind.MutatingWebhookConfiguration | kind.ValidatingWebhookConfiguration,
```

Comment:
```suggestion
  webhookConfiguration: kind.MutatingWebhookConfiguration | kind.ValidatingWebhookConfiguration,
```

Abbreviated parameter names are worth avoiding, even though one could reasonably infer the meaning of `whc` thanks to the typing provided.

Reply:
`Abbreviated parameter names are worth avoiding`

Inside small-ish functions (<100 lines) with well-defined types, I haven't found this to be particularly valuable -- I actually find expansive variable names that simply restate types to be rather _harder_ to scan over, personally.

But willing to go on an adventure with you on this one. :man_shrugging: